### DISCUSSION TITLE
Is there a way to disable CRAM-MD5 in postal?

### DISCUSSION THREAD

**Discussion by gitbnitin** [postalserver/postal#2888] [Help with using Postal]
Is there a way to disable CRAM-MD5 in postal?

Apollo.io doesn't support CRAM-MD5, so we would like to disable CRAM-MD5 on one of our server.

**Comment by catphish** [comment]
Can you explain why having CRAM-MD5 enabled would be a problem in this case? Clients should simply ignore protocols they don't support.

**Reply by gitbnitin** [reply]
Apollo.io seems to have some issue, and they are saying that if we can disable CRAM-MD5, their application will choose plain authentication and the workflow will work fine.

**Reply by catphish** [reply]
I think it would be better to try to address this issue properly. If they didn't support CRAM-MD5 then they wouldn't be attempting to use it. The fact that they are attempting to use it suggests that they do support it and that there is a compatibility problem. Please can you ask them for further information about the nature of the problem?

**Reply by gitbnitin** [reply]
They are clearly saying that their client has some issue that if CRAM-MD5 fails, ideally it should fallback to plain auth, but it is not. so they are accepting their problem, but as a workaround, they are asking if I can disable CRAM-MD5 in SMTP server side (postal side), so that the capability negotiation will work fine and then once they apply some solution, CRAM-MD5 can be re-enabled. Their bug fix time seems to be six months.

**Reply by catphish** [reply]
Thanks. That's annoying. I just wanted to make sure that if it was our issue, we fixed it.
